src/Transactions/TransactionsList/TransactionsFilters/ExpenseClassFilter/ExpenseClassFilterContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { expenseClassesManifest } from '@folio/stripes-acq-components';
+
+import ExpenseClassFilter from './ExpenseClassFilter';
+
+const ExpenseClassFilterContainer = ({ resources, ...rest }) => {
```

Comment:
resources?

Reply:
and where is manifest for it?

Reply:
It declared below (please look at 24th line :)

Reply:
but why via resources? is there loading of expense classes somewhere else to avoid additional call?

Reply:
Now on transaction list expense class filter uses expense class collection only.